PlayerToken.java

```diff
@@ -0,0 +1,50 @@
+package game.logic;
+enum PlayerToken { NOPLAYER, PLAYER1, PLAYER2 };
+
+
+private class Player {
+	
+	private PlayerToken playerTokens;
+	
+	
+	private int numPieces;
+	
+	private int numPiecesOnBoard;
+	
+	private int numPiecesLeft;
+	
+	
+	
```

Comment:
There should be a constructor here, that way we can set the player token and the number of pieces when the Player object is created.

It would be expected to call the Player object like so
```java
# where 9 represents the number of pieces the player has
Player p1 = new Player(PlayerToken.PLAYER1, 9);
```